Frontend/components/statsCount.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import CountUp from "react-countup";
+
+function StatsCount() {
+  return (
+    <div className="grid grid-cols-1 md:grid-cols-3 gap-4 mt-10">
+      <div className="text-center">
+        <div style={{ fontSize: "25px" }} className="font-bold">
+          <CountUp start={0} end={5000} duration={3} />
+          +
+        </div>
+        <div className="font-bold mt-2">
+          <p>Active members</p>
+        </div>
+      </div>
+      <div className="text-center mb-8 sm:mb-2">
+        <div style={{ fontSize: "25px" }} className="font-bold">
+          <CountUp start={0} end={300} duration={3} />
+          +
+        </div>
+        <div className="font-bold mt-2">
+          <p>Community events organised</p>
+        </div>
```

Comment:
![Screen Shot 2023-11-13 at 11 36 32](https://github.com/DevsKonnekt/DevsKonnekt/assets/63302923/0b723ec0-b924-4f17-8c14-ca18c04d1c38)

Reply:
I have resolved this one

Reply:
Okay thank you

Reply:
Will be pushing the changes any minute